zerver/lib/bugdown/__init__.py

```diff
@@ -1179,7 +1179,7 @@ def handleMatch(self, match: Match[str]) -> Optional[Element]:
         elif name in name_to_codepoint:
             return make_emoji(name_to_codepoint[name], orig_syntax)
         else:
-            return None
+            return orig_syntax
```

Comment:
It is probably of interest that this was a deliberate change in 88423496294cf6c01ecc258acf01a203f600d570.

We've since migrated that testcase from that commit to markdown_test_cases and it works as expected for this commit (which is probably because linkifiers run before emojifiers do).

Reply:
@aero31aero so, are you suggesting that we should look for a better solution?